tools/theme/README.md

```diff
@@ -54,9 +51,49 @@ import '@spectrum-web-components/theme/express/scale-medium.js';
 import '@spectrum-web-components/theme/express/scale-large.js';
 ```
 
+When looking to leverage the `Theme` base class as a type and/or for extension purposes, do so via:
+
+```
+import { Theme } from '@spectrum-web-components/theme';
+```
+
+## What it's doing
+
+Visually, all Spectrum Web Components are an expression of the design tokens specified by Spectrum, Adobe's design system. On the web, these tokens are applied as CSS Custom Properties. Using `sp-theme` as a parent element for your components ensures that those elements' styles are not only consistent with the color and scale you've specified in your particular instance, but for _all_ the other color, scale, and content direction specifications across Spectrum.
+
+Additionally, the `sp-theme` element manages the content direction applied to the elements in its DOM scope. By default, an `sp-theme` element resolves its initial content direction from the value of its `dir` attribute; otherwise, it matches the specification of the containing `sp-theme` parent or `document`. Subsequent customization of content direction for its children must occur on the `sp-theme` element itself, [REVIEW ME bc idk if this is accurate] either through an inline `style` attribute or in a CSS declaration [END REVIEW ME], to be appropriately tracked by elements in that scope. This means that each part of your document scoped by an `sp-theme` element can specify its own content direction, and that descendent `sp-theme` elements can override the content direction applied by its ancestor elements.
```

Comment:
Please take a look at this line!! 

> either through an inline `style` attribute or in a CSS declaration

I want to make sure this statement in particular is accurate

Reply:
This _is_ complex...by DEFAULT, `sp-theme` resolves its `dir` value to its first `sp-theme` or `document` ancestor, or to `ltr` if on is not present there: https://github.com/adobe/spectrum-web-components/blob/main/tools/theme/src/Theme.ts#L398-L413

When `sp-theme` is manually supplied a value for `dir` this value overrides the default.

In all cases, the `sp-theme` will manage the `dir` value of any `SpectrumElement` decedents where there is not an intervening `sp-theme` element.

This can only happen directly on the attribute/property itself but will effect the way that CSS is applied across the system as many Spectrum CSS rules are still built around `[dir="lrt"] ...` or `[dir="rtl"] ...`. We are working to remove that in favor of logical properties, at which point we will likely look to remove this management as it does have a performance cost...